integrationtests/tests/fixtures/mod.rs

```diff
@@ -1020,60 +1038,59 @@ impl FederationTest {
         Ok(())
     }
 
+    /// Does any of the modules return consensus proposal that forces a new epoch
     #[allow(clippy::await_holding_refcell_ref)]
     pub async fn has_pending_epoch(&self) -> bool {
         for server in &self.servers {
-            if server
+            let mut server = server.borrow_mut();
+            let fedimint_server = &mut server.fedimint;
+
+            // Pending transactions will trigger an epoch
+            if let Ok(tx) = fedimint_server.tx_receiver.get_mut().as_mut().try_recv() {
+                fedimint_server
+                    .consensus
+                    .tx_sender
+                    .send(tx)
+                    .await
+                    .expect("Can send");
+                return true;
+            }
+
+            let consensus_proposal = fedimint_server.consensus.get_consensus_proposal().await;
+            if !consensus_proposal.force_new_epoch {
+                continue;
+            }
+
+            if !consensus_proposal.items.is_empty() {
+                return true;
+            }
```

Comment:
Is this still correct? Shouldn't there just always be an epoch if `force_new_epoch == true`? It will never be a problem currently because the wallet always contributes opportunistically, but if that ever changes this might lead to some annoying debugging (e.g. rejoin-epochs might be empty but will be forced).

Reply:
Agreed. Fixed.